Using Git.md

```diff
@@ -60,6 +61,16 @@ git checkout feature/login-screen
 git merge dev
 ```
 
+### Release branches
+
+Most of the projects have one active stream at the moment. This means that all of the work done by developer can go directly into the dev branch. All the work goes into the next release so there is no need to seperate the dev branch into mulitple release branches. 
+
+However, some projects have multiple releases plan ahead. It is possible that at some point a development team will work on multiple releases at the same time. In that case, `release` branches should be introduced. Protect this branches as you do for your main branch. You can use regex to match all the release branches and not worry about protecting a newly create ones. **Note:** If you want to delete a protected branch on Github you will need to remove the protection cause this can't be done even with admin privilege. 
```

Comment:
```suggestion
However, some projects have multiple releases planned ahead. It is possible that at some point a development team will work on multiple releases at the same time. In that case, `release` branches should be introduced and handled in the same manner as the main `dev` branch. You can use regex to match all the release branches and not worry about protecting newly created ones. **Note:** If you want to delete a protected branch on GitHub you will need to remove the protection because this can't be done even with admin privileges. 
```